.clang-format

```diff
@@ -42,9 +42,9 @@ AllowAllParametersOfDeclarationOnNextLine: true
 AllowShortBlocksOnASingleLine: Never
 AllowShortCaseLabelsOnASingleLine: false
```

Comment:
@kreuzerkrieg please check the changes in the .clang-format if you agree (see the other comments about the format putting quite long functions on a single line)

wondering if it would actually make sense to make a shorter line column limit instead ...